docs/docsite/rst/network/getting_started/sample_files/first_playbook.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: Network Getting Started: First Playbook
```

Comment:
`mapping values are not allowed here` generally means you've got an extra `:`

How about
`name: Network Getting Started - First Playbook` - will need to update references in doc

OR
you could quote the string, but then it would be strange that you don't quote this. Also if someone forgets the quotes they will get a similar error message and also be confused

http://docs.ansible.com/ansible/latest/YAMLSyntax.html#gotchas